pkgs/by-name/ga/galleta/package.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, fetchzip
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "galleta";
+  version = "20040505_1";
+
+  src = fetchzip {
+      url = "http://sourceforge.net/projects/fast/files/Galleta/Galleta%20v${finalAttrs.version}/galleta_${finalAttrs.version}.zip";
+      hash = "sha256-tc5XLToyQZutb51ZoBlGWXDpsSqdJ89bjzJwY8kRncA=";
+  };
+
+  makeFlags = [
+    "-C src"
+  ];
+
+  enableParallelBuilding = true;
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp bin/galleta $out/bin
+  '';
+
+  meta = with lib; {
+    description = "Examine the contents of the IE's cookie files for forensic purposes";
+    mainProgram = "galleta";
+    homepage = "https://sourceforge.net/projects/odessa/files/Galleta";
+    maintainers = with maintainers; [ d3vil0p3r ];
+    platforms = with platforms; linux ++ darwin;
+    license = licenses.gpl2;
```

The file header of `galleta.c` looks like the content of the 3-Clause BSD license.

 ```suggestion
     license = licenses.bsd3;
```